src/renderer/page/publish/index.js

```diff
@@ -18,34 +15,40 @@ import {
 } from 'redux/actions/publish';
 import PublishPage from './view';
 
-const select = (state, props) => {
+const select = state => {
+  const isStillEditing = selectIsStillEditing(state);
+  const myClaimForUri = selectMyClaimForUri(state);
   const publishState = selectPublishFormValues(state);
-  const { uri, name } = publishState;
+  const { uri } = publishState;
 
   const resolvingUris = selectResolvingUris(state);
   let isResolvingUri = false;
   if (uri) {
     isResolvingUri = resolvingUris.includes(uri);
   }
 
-  const claimsByUri = selectClaimsByUri(state);
-  const myClaims = selectMyClaims(state);
-
-  const claimForUri = claimsByUri[uri];
+  let claimForUri;
   let winningBidForClaimUri;
-  let myClaimForUri;
-  if (claimForUri) {
-    winningBidForClaimUri = claimForUri.effective_amount;
-    myClaimForUri = myClaims.find(claim => claim.name === name);
+  if (!myClaimForUri) {
+    // if the uri isn't from a users claim, find the winning bid needed for the vanity url
+    // in the future we may want to display this on users claims
+    // ex: "you own this, for 5 more lbc you will win this claim"
+    const claimsByUri = selectClaimsByUri(state);
+    claimForUri = claimsByUri[uri];
+    winningBidForClaimUri = claimForUri ? claimForUri.effective_amount : undefined;
```

Comment:
`null` seems more correct than `undefined` - we know there's not one; we're not implying it will be set later